en/src/pages/index.module.css

```
@@ -8,6 +8,58 @@
+  flex-shrink: 0;
+  position: sticky;
+  top: var(--ifm-navbar-height);
+  height: calc(100vh - var(--ifm-navbar-height));
+  background-color: var(--wso2-sidebar-bg);
+  border-right: 1px solid var(--wso2-surface-border);
+  z-index: 10;
+  overflow-y: auto;
+}
+
+/* Add a small top padding for a premium, balanced look */
+.sidebarCol > :global(div) {
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)